examples/rbac/feature_store_remote.yaml

```diff
@@ -0,0 +1,28 @@
+project: server
+provider: local
+registry:
+    registry_type: sql
+    path: postgresql://feast:feast@postgresql.feast-qe.svc.cluster.local:5432/feast
+    cache_ttl_seconds: 60
+    sqlalchemy_config_kwargs:
+        echo: false
+        pool_pre_ping: true
+online_store:
+    type: postgres
+    host: postgresql.feast-qe.svc.cluster.local
+    port: 5432
+    database: feast
+    db_schema: public
+    user: feast
+    password: feast
+offline_store:
+    type: postgres
+    host: postgresql.feast-qe.svc.cluster.local
+    port: 5432
+    database: feast
+    db_schema: public
+    user: feast
+    password: feast
+auth:
+    type: kubernetes
```

Comment:
do user required to mention the auth setting both side server and client ?

Reply:
I think so, we are referring auth configuration on the server side as well.

Reply:
I think so as well